<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>ServerWindow</class>
    <widget class="QWidget" name="ServerWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>600</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <widget class="QWidget" name="centralwidget" native="true">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>10</y>
                    <width>801</width>
                    <height>581</height>
                </rect>
            </property>
            <widget class="QTextBrowser" name="serverStatus">
                <property name="geometry">
                    <rect>
                        <x>10</x>
                        <y>20</y>
                        <width>781</width>
                        <height>481</height>
                    </rect>
                </property>
                <property name="contextMenuPolicy">
                    <enum>Qt::ActionsContextMenu</enum>
                </property>
            </widget>
            <widget class="QPushButton" name="serverStart">
                <property name="geometry">
                    <rect>
                        <x>300</x>
                        <y>540</y>
                        <width>75</width>
                        <height>23</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Start</string>
                </property>
            </widget>
            <widget class="QPushButton" name="serverStop">
                <property name="geometry">
                    <rect>
                        <x>420</x>
                        <y>540</y>
                        <width>75</width>
                        <height>23</height>
                    </rect>
                </property>
                <property name="text">
                    <string>Stop</string>
                </property>
            </widget>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>800</width>
                    <height>21</height>
                </rect>
            </property>
        </widget>
        <widget class="QStatusBar" name="statusbar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>3</width>
                    <height>18</height>
                </rect>
            </property>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
